asm/daxpy_amd64.s

```diff
@@ -136,3 +136,67 @@ V2:
 
 E2:
 	RET
+
+// func DaxpyIncTo(dst []float64, incDst, idst uintptr, alpha float64, x, y []float64, n, incX, incY, ix, iy uintptr)
+TEXT ·DaxpyIncTo(SB), NOSPLIT, $0
+	MOVQ   dst+0(FP), R10
+	MOVQ   incDst+24(FP), R13
+	MOVQ   idst+32(FP), BP
+	MOVHPD alpha+40(FP), X7
+	MOVLPD alpha+40(FP), X7
+	MOVQ   x+48(FP), R8
+	MOVQ   y+72(FP), R9
+	MOVQ   n+96(FP), DX
+	MOVQ   incX+104(FP), R11
+	MOVQ   incY+112(FP), R12
+	MOVQ   ix+120(FP), SI
+	MOVQ   iy+128(FP), DI
+
+	MOVQ SI, AX  // nextX = ix
```

Comment:
I take it that the code here was copy/past from DaxpyInc except for the preamble? Can we have either a comment to this effect or something that generates both?